src/components/accordion/accordion.ts

```diff
@@ -0,0 +1,134 @@
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { defineComponents } from '../common/definitions/defineComponents.js';
+import IgcExpansionPanelComponent from '../expansion-panel/expansion-panel.js';
+
+defineComponents(IgcExpansionPanelComponent);
+
+/**
+ * The Accordion is a container-based component that can house multiple expansion panels
+ * and offers keyboard navigation.
+ *
+ * @element igc-accordion
+ *
+ * @slot - Renders the expansion panels inside default slot.
+ */
+export default class IgcAccordionComponent extends LitElement {
+  public static readonly tagName = 'igc-accordion';
+  //public static styles = styles;
+
+  @property({ reflect: true, type: Boolean })
+  public singleBranchExpand = false;
```

Comment:
Branch sounds a bit odd to me in the accordion actually (implies hierarchy I think) so perhaps we can rename it to just `singleExpand`.
And the property decorator should state the explicit DOM attribute with `attribute: 'single-expand'`